src/app/lib/pages/map_page/map_page.dart

```diff
@@ -29,12 +32,25 @@ class MapPageState extends State<MapPage> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-        body: GoogleMap(
-            onMapCreated: _onMapCreated,
-            markers: _markers,
-            initialCameraPosition: CameraPosition(
-                target: LatLng(43.55103829955488, -79.66262838104547),
-                zoom: 15)));
+      floatingActionButton: FloatingActionButton(
+        onPressed: () {
+          Route route =
+              MaterialPageRoute(builder: (context) => AddEvent(userCreds: widget.userCreds,));
+          Navigator.push(context, route);
+        },
+        child: const Icon(
+          Icons.add,
+          color: Colors.blue,
+        ),
+        backgroundColor: Colors.white,
+      ),
+      body: GoogleMap(
+          zoomControlsEnabled: false,
+          onMapCreated: _onMapCreated,
+          markers: _markers,
+          initialCameraPosition: const CameraPosition(
+              target: LatLng(43.55103829955488, -79.66262838104547), zoom: 15)),
```

Comment:
<img width="156" alt="Screen Shot 2021-10-20 at 5 21 48 PM" src="https://user-images.githubusercontent.com/36747253/138174094-32ce770d-32d0-4c1f-a23b-d262918b982b.png">
 
On ios, a current location button will be shown and it overlaps with the add event button. I think it would be a good idea to keep it so we will turn it on explicitly by adding the option
```
myLocationButtonEnabled: false,
```
We would also want to move the location of the add event to be above of the location button.